### DISCUSSION TITLE
buddhism/2023/books/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#33] [General]
# buddhism/books/

https://shanenull.com/buddhism/2023/books/

**Comment by shane0** [comment]
https://www.ctworld.org.tw/Buddhist%20e-Books/Books05/index.html

**Comment by shane0** [comment]
https://www.buddhajewel.org/en/sutras/

**Comment by shane0** [comment]
paths to enlightenment with more details on week 9 buddhist vegetarian
https://www.ctworld.org.tw/Buddhist%20e-Books/Audio/Book004/index.html

q&a: leather clothes: maybe if sourced without suffering but no reason because you can use cotton or other plants, eggs: are basically a baby chicken not that subtle, expedient means or gradual breaking habits is a valid path, 3 fold path on page 69

**Comment by shane0** [comment]
reading zen reader

**Comment by shane0** [comment]
zfzb 101

https://youtu.be/Y0p663Ot8mo?si=BnUzi-CrqtqSGh3M
